konnect-runtime-setup.sh

```diff
@@ -122,6 +126,12 @@ check_variables() {
         error "Konnect runtime image name is missing"
     fi
 
+        # temporary fix for multiplatform support
+    if [[ "${VERSIONS_TO_APPLY_ALPINE_SUFFIX[*]}" =~ ${KONNECT_RUNTIME_IMAGE} ]]; then
```

Comment:
nit: spacing and add more context to the comment; even if this is temporary we should state the reasons for this one (has ramifications on user experience).

```suggestion
    # fix for enabling multi-platform support to Kong Gateway image versions
    if [[ "${VERSIONS_TO_APPLY_ALPINE_SUFFIX[*]}" =~ ${KONNECT_RUNTIME_IMAGE} ]]; then
```